spec/features/user/invalid_path_page_not_found_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe "invalid path" do
+  it "directs to page not found" do
+    visit '/errors/file_not_found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.